fixed typos in function.json for Webhook output

panjkov opened the pull request:
Fixed invalid type and subscriptionResource attribute names and values in function.json for both csx and js examples for Webhook Output

azurecla commented:
Hi __@slapointe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

tysonn commented:
@seanmck - Sean, could you review these updates proposed by @slapointe in the public repo?

seanmck commented:
Already fixed - you can close.

tysonn commented:
@slapointe - Thanks for the update - I apologize for the delayed review. Closing per the author's feedback.

panjkov committed:
fixed typos in function.json for Webhook output

Fixed invalid type and subscriptionResource attribute names and values in function.json for both csx and js examples for Webhook Output
